### ISSUE TITLE
Error when updating a user's password from the administration

### ISSUE THREAD

**Issue by kevin-DL** [supertokens/supertokens-golang#311]
Context : 
  - Supertokens Core 5
  - SDK 0.12.8

When i try to update the password of a registered user through the admin dashboard, i get the following error:

```bash
2023/07/25 10:57:53 http: panic serving 192.168.0.2:39048: runtime error: invalid memory address or nil pointer dereference
﻿goroutine 66 [running]:
﻿net/http.(*conn).serve.func1()
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/net/http/server.go:1850 +0xbf
﻿panic({0xc35bc0, 0x147a960})
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/runtime/panic.go:890 +0x262
﻿github.com/honeybadger-io/honeybadger-go.(*Client).Handler.func1.1()
﻿/root/go/pkg/mod/github.com/honeybadger-io/honeybadger-go@v0.5.0/client.go:102 +0x1e5
﻿panic({0xc35bc0, 0x147a960})
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/runtime/panic.go:884 +0x212
﻿github.com/supertokens/supertokens-golang/recipe/dashboard/api/userdetails.UserPasswordPut({_}, {{0xc000134bf0, 0xc000134bf8}, {{0xc000038051, 0xa}, {{0xc00034a8b8, 0x18}}, {{0xc000041f20, 0x1c}}, {0xc000041f2c, ...}, ...}, ...})
﻿/root/go/pkg/mod/github.com/supertokens/supertokens-golang@v0.12.8/recipe/dashboard/api/userdetails/userPasswordPut.go:131 +0x1a5
﻿github.com/supertokens/supertokens-golang/recipe/dashboard.(*Recipe).handleAPIRequest.func1()
﻿/root/go/pkg/mod/github.com/supertokens/supertokens-golang@v0.12.8/recipe/dashboard/recipe.go:166 +0xd58
﻿github.com/supertokens/supertokens-golang/recipe/dashboard.apiKeyProtector({_}, {{0xc000134bf0, 0xc000134bf8}, {{0xc000038051, 0xa}, {{0xc00034a8b8, 0x18}}, {{0xc000041f20, 0x1c}}, {0xc000041f2c, ...}, ...}, ...}, ...)
﻿/root/go/pkg/mod/github.com/supertokens/supertokens-golang@v0.12.8/recipe/dashboard/apiKeyProtector.go:18 +0x89
﻿github.com/supertokens/supertokens-golang/recipe/dashboard.(*Recipe).handleAPIRequest(0xc00023a900, {0xd3a536, 0x12}, 0xc000116320?, {0x7f4ea1c61458?, 0xc0003e6090?}, 0xc5b900?, {{0xc807a0?, 0x0?}}, {0xc0003dad20, ...})
﻿/root/go/pkg/mod/github.com/supertokens/supertokens-golang@v0.12.8/recipe/dashboard/recipe.go:122 +0x386
﻿github.com/supertokens/supertokens-golang/supertokens.(*superTokens).middleware.func2({0x7f4ea1c5fb38?, 0xc00062c200}, 0xc000048700)
﻿/root/go/pkg/mod/github.com/supertokens/supertokens-golang@v0.12.8/supertokens/supertokens.go:207 +0x7da
﻿net/http.HandlerFunc.ServeHTTP(0xc000307860?, {0x7f4ea1c5fb38?, 0xc00062c200?}, 0xc000048700?)
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/net/http/server.go:2109 +0x2f
﻿github.com/go-chi/cors.(*Cors).Handler.func1({0x7f4ea1c5fb38, 0xc00062c200}, 0xc000048700)
﻿/root/go/pkg/mod/github.com/go-chi/cors@v1.2.1/cors.go:228 +0x1bd
﻿net/http.HandlerFunc.ServeHTTP(0xc000048300?, {0x7f4ea1c5fb38?, 0xc00062c200?}, 0xd0?)
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/net/http/server.go:2109 +0x2f
﻿github.com/go-chi/chi/v5/middleware.RequestLogger.func1.1({0xfbbc70, 0xc0000f0000}, 0xc000048300)
﻿/root/go/pkg/mod/github.com/go-chi/chi/v5@v5.0.8/middleware/logger.go:54 +0x17d
﻿net/http.HandlerFunc.ServeHTTP(0xfbc358?, {0xfbbc70?, 0xc0000f0000?}, 0x147a5f0?)
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/net/http/server.go:2109 +0x2f
﻿github.com/go-chi/chi/v5.(*Mux).ServeHTTP(0xc000371e60, {0xfbbc70, 0xc0000f0000}, 0xc000048100)
﻿/root/go/pkg/mod/github.com/go-chi/chi/v5@v5.0.8/mux.go:90 +0x310
﻿github.com/honeybadger-io/honeybadger-go.(*Client).Handler.func1({0xfbbc70?, 0xc0000f0000?}, 0xc0003e2150?)
﻿/root/go/pkg/mod/github.com/honeybadger-io/honeybadger-go@v0.5.0/client.go:105 +0x85
﻿net/http.HandlerFunc.ServeHTTP(0x0?, {0xfbbc70?, 0xc0000f0000?}, 0x6fd3d4?)
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/net/http/server.go:2109 +0x2f
﻿net/http.serverHandler.ServeHTTP({0xfba7b8?}, {0xfbbc70, 0xc0000f0000}, 0xc000048100)
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/net/http/server.go:2947 +0x30c
﻿net/http.(*conn).serve(0xc0003fe000, {0xfbc400, 0xc000518570})
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/net/http/server.go:1991 +0x607
﻿created by net/http.(*Server).Serve
﻿/nix/store/a7875alzpnr46z6mv4ssymfdwmvr6xbq-go-1.19.4/share/go/src/net/http/server.go:3102 +0x4db```

**Comment by rishabhpoddar** [comment]
This has been fixed in golang SDK version >= 0.12.9

**Comment by kevin-DL** [comment]
Thank you, it's working fine now.
